prowler/lib/outputs/json.py

```diff
@@ -70,12 +85,147 @@ def fill_json_asff(finding_output, audit_info, finding, output_options):
     return finding_output
 
 
+def fill_json_ocsf(
+    finding_output: Check_Output_JSON_OCSF, audit_info, finding, output_options
+):
+    resource_region = ""
+    resource_name = ""
+    resource_uid = ""
+    finding_uid = ""
+    resource_labels = finding.resource_tags if finding.resource_tags else []
+    if finding.status == "PASS":
```

Comment:
Also you should split this function into several ones to fill the model, it's hard to follow it.